### PR TITLE
fix tests for IE11

### PR DESCRIPTION
**Pull request by cherifGsoul**
* Fix routing values conflicts between tests with timeout for ie11.
* Skip `can-observe` tests

### CODE REVIEW

**Comment by matthewp** [comment]
lgtm

File: test/route-define-test.js [hunk 1]
```diff
@@ -61,14 +61,16 @@ if (typeof steal !== 'undefined') {
...
+			setTimeout(function(){
+				equal(mockRoute.hash.value, "cat/5&section=home", "same URL");
+				equal(appState.get("type"), "cat", "hash populates the appState");
+				equal(appState.get("id"), "5", "hash populates the appState");
+				equal(appState.get("section"), "home", "appState keeps its properties");
+				ok(canRoute.data === appState, "canRoute.data is the same as appState");
+	
+				mockRoute.stop();
+				QUnit.start();
+			},0.5);
```
Comment (justinbmeyer) [inline test/route-define-test.js]: Can you explain this?  It's __very__ good practice to put inline comments describing any changes that someone would not understand.

Comment (phillipskevin) [reply test/route-define-test.js]: _If_ this change is needed, it really should be done inside `_onStartComplete`. The purpose of that function is to tell tests when they can do assertions.

That said, I still don't understand why this change is needed. Why is `_onStartComplete` being called before things have settled?

**Comment by phillipskevin** [comment]
I think this should be ready to go now, @cherifGsoul. There are still a few flaky tests that we need to fix, but they are not IE11 specific. I see them failing intermittently in Chrome also.

**Comment by phillipskevin** [comment]
test-saucelabs is failing. I'll take a look in a bit.

**Pull request merged**